<ui version="4.0" >
 <class>SPEA2ModelerWidget</class>
 <widget class="QWidget" name="SPEA2ModelerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Number of discretization steps</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="sliderDSteps" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QSpinBox" name="numdstepsEdit" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Number of states</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="sliderStates" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QSpinBox" name="numstatesEdit" >
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Size of Population</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSlider" name="sliderPopSize" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QSpinBox" name="popSize" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Size of Archive</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSlider" name="sliderArchiveSize" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QSpinBox" name="archiveSize" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Number of Iterations</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QSlider" name="sliderNumIterations" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QSpinBox" name="numIterations" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Number of Autocorrelation coefficients to consider</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QSlider" name="sliderAutocorrelation" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="3" >
    <widget class="QSpinBox" name="autocorrelation" />
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Probability of "Crossover" Mutator</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" >
    <widget class="QDoubleSpinBox" name="mutCrossover" >
     <property name="maximum" >
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
     <property name="value" >
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Probability of "Block" Mutator</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" >
    <widget class="QDoubleSpinBox" name="mutBlock" >
     <property name="maximum" >
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
     <property name="value" >
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>Probability of "Reverse" Mutator</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" >
    <widget class="QDoubleSpinBox" name="mutInverse" >
     <property name="maximum" >
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
     <property name="value" >
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <widget class="QLabel" name="label_10" >
     <property name="text" >
      <string>Probability of "Shuffle" Mutator</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" >
    <widget class="QDoubleSpinBox" name="mutShuffle" >
     <property name="minimum" >
      <double>0.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
     <property name="value" >
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="0" >
    <widget class="QLabel" name="label_11" >
     <property name="text" >
      <string>Probability of "Swap" Mutator</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3" >
    <widget class="QDoubleSpinBox" name="mutSwap" >
     <property name="maximum" >
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.010000000000000</double>
     </property>
     <property name="value" >
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>393</width>
       <height>201</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2" >
    <widget class="QPushButton" name="computeButton" >
     <property name="text" >
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2" >
    <widget class="QLabel" name="progressLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderDSteps</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numdstepsEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>432</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numdstepsEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderDSteps</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>421</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>279</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderStates</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numstatesEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>280</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numstatesEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderStates</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>412</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>276</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>popSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderPopSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>442</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel" >
     <x>280</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderPopSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>popSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>207</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel" >
     <x>419</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderArchiveSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>archiveSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>247</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel" >
     <x>424</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>archiveSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderArchiveSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>411</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel" >
     <x>285</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderNumIterations</sender>
   <signal>valueChanged(int)</signal>
   <receiver>numIterations</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>267</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel" >
     <x>427</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numIterations</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderNumIterations</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>417</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel" >
     <x>295</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderAutocorrelation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>autocorrelation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>275</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel" >
     <x>434</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autocorrelation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderAutocorrelation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>415</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel" >
     <x>281</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
